Clinical-Genomics/scout
# Select preferably users with _id defined by an email

This might (or might not ) fix a number of bugs. I don't have much time to test this right now. If someone wants they could take over!

fix #1472
fix #1481
fix #1480
fix #1476

**How to test**:
1. Test on Scout stage using a user with a double account: one with _id of type ObjectID and one with _id as an email address.
1. Make sure that the user object with most institutes is selected upon login (should be the one with _id = email address)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
